tests/testthat/test-bundle.R

```diff
@@ -569,3 +571,29 @@ test_that("writeManifest: Sets environment.image in the manifest if one is provi
   manifest <- makeManifest(appDir, appPrimaryDoc = NULL)
   expect_null(manifest$environment)
 })
+
+test_that("tarImplementation: checks environment variable and option before using default", {
+  # Environment variable only set should use environment varaible
+  Sys.setenv("RSCONNECT_TAR" = "envvar")
```

Comment:
Is there any point in restoring any incoming `RSCONNECT_TAR` or `rsconnect.tar` values on the way out of the test?

Reply:
Adjusted so that this happens.